frontend/src/app/patients/ManagePatients.tsx

```diff
@@ -88,7 +159,9 @@ const ManagePatients = ({ activeFacilityId, canEditUser }: Props) => {
         <div className="grid-row">
           <div className="prime-container usa-card__container">
             <div className="usa-card__header">
-              <h2>{PATIENT_TERM_PLURAL_CAP}</h2>
+              <h2>
+                {PATIENT_TERM_PLURAL_CAP}{pageCount > 0 ? ` (Page {currentPage} of {pageCount})` : ""}
+              </h2>
```

Comment:
![Screen Shot 2021-02-26 at 9 46 33 AM](https://user-images.githubusercontent.com/53869143/109314699-ae5f6380-7817-11eb-9311-16913d4eca24.png)

The prototype has text "Showing X of Y" not "Page X of Y". Also the text following the header needs to be a different size and color. The suggestion below would get closer to the mock but you will need to add a class to the paragraph for the styling. I would also ask @hmyers-cms and @katiealoisi how the "Showing X of Y" should change per page

```suggestion
              <div>
                <h2>{PATIENT_TERM_PLURAL_CAP}</h2>
                {pageCount > 0 ? <p> (Showing {entriesPerPage} of {totalPatients.patientsCount})</p> : ""}
              </div>
```